pkg/controller/common/cni/cni.go

```diff
@@ -33,15 +36,17 @@ func InitConfig(m manager.Manager) (Config, error) {
 	} else {
 		log.Info(fmt.Sprintf("CNI is enabled for this installation: %v", config.Enabled))
 	}
+
+	config.Enabled = true
```

Comment:
do we want to enable CNI even if multus is not in use?

Reply:
Yes. Multus simply allows us to support multiple Istio CNI versions at the same time (and fixes the race conditions).